extensions/cornerstone/src/utils/isMeasurementWithinViewport.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Determines if a measurement is within the current viewport extent.
+ * Checks if all measurement points fall within the visible area defined by the camera's
+ * focal point and parallel scale (extent boundaries).
+ *
+ * @param {Object} params - The parameters object
+ * @param {Object} params.viewport - The viewport object containing camera information
+ * @param {Object} params.measurement - The measurement object containing points to check
+ * @param {Array<Array<number>>} params.measurement.points - Array of 3D points [x, y, z]
+ * @returns {boolean} True if all measurement points are within the viewport extent, false otherwise
+ */
+export const isMeasurementWithinViewport = (viewport, measurement) => {
```

Comment:
This could use the getMeasurementExtent, and then just test on the extent function.